glom/tutorial.py

```diff
@@ -132,25 +182,40 @@
 Unfortunately, data in the real world is messy. You might be expecting a certain format and end up getting something
 completely different. No worries, glom to the rescue.
 
-Coalesce is a glom construct that allows you to specify fallback behavior for a list of subspecs.
+**Coalesce** is a glom construct that allows you to specify fallback behavior for a list of subspecs.
 Subspecs are passed as positional arguments, while defaults can be set using keyword arguments.
 
 Let's say our astronomers recently got a new update in their systems, and sometimes ``system`` will contain
 ``dwarf_planets`` instead of ``planets``.
 
 To handle this, we can define the ``dwarf_planets`` subspec as a Coalesce fallback.
 
-  >>> target = {'system': {'planets': [{'name': 'earth', 'moons': 1},
-  ...                                  {'name': 'jupiter', 'moons': 69}]}}
-  >>> spec = {'names': (Coalesce('system.planets', 'system.dwarf_planets'), ['name']),
-  ...         'moons': (Coalesce('system.planets', 'system.dwarf_planets'), ['moons'])}
+  >>> from glom import Coalesce
+  >>> target = {
+  ...     'system': {
+  ...         'planets': [
+  ...             {'name': 'earth', 'moons': 1},
+  ...             {'name': 'jupiter', 'moons': 69}
+  ...         ]
+  ...     }
+  ... }
+  >>> spec = {
+  ...     'names': (Coalesce('system.planets', 'system.dwarf_planets'), ['name']),
```

Comment:
I've been thinking, should "names" here be "planets" instead? I guess they're technically Planetary Mass Objects (planemos :P) https://en.wikipedia.org/wiki/Planet#Planetary-mass_objects

The things you learn when writing docs!

Reply:
i agree that "planets" makes more sense than names here 👍